include/sound/sof/info.h

```diff
@@ -43,6 +54,7 @@ struct sof_ipc_fw_version {
 	uint8_t time[10];
 	uint8_t tag[6];
 	uint32_t abi_version;
+	struct sof_ipc_cc_version compiler;
```

Comment:
doesn't this break backwards compatibility?

Reply:
outdated